src/main/io/osd.c

```diff
@@ -1146,6 +1146,53 @@ static bool osdDrawSingleElement(uint8_t item)
     return true;
 }
 
+#ifdef USE_OSD_STICK_OVERLAY
+static void osdDrawStockOverlayAxis(uint8_t xpos, uint8_t ypos)
+{
+
+    for (uint8_t x = 0; x < OSD_STICK_OVERLAY_WIDTH; x++) {
+        for (uint8_t y = 0; y < OSD_STICK_OVERLAY_HEIGHT; y++) {
+            // draw the axes, vertical and horizonal
+            if (x == ((OSD_STICK_OVERLAY_WIDTH-1)/2)) {
+                displayWriteChar(osdDisplayPort, xpos+x, ypos+y, osdConfig()->overlay_vertical_char);
+            }
+            if (y == ((OSD_STICK_OVERLAY_HEIGHT-1)/2)) {
+                displayWriteChar(osdDisplayPort, xpos+x, ypos+y, osdConfig()->overlay_horizontal_char);
+            }
+            if ((x == ((OSD_STICK_OVERLAY_WIDTH-1)/2)) && (y == (OSD_STICK_OVERLAY_HEIGHT-1)/2)) {
```

Comment:
You can put this first and use `else` branch to draw lines, avoiding overwriting center char